.github/workflows/universal_decoder_ci.yaml

```diff
@@ -71,13 +77,17 @@ jobs:
       - name: Get the image version
         id: vars
         run: echo ::set-output name=tag::0.0.$(echo ${GITHUB_SHA})
+      - name: Login to docker
+        run: docker login -u ${{ secrets.DOCKER_LOGIN }} -p ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
This step should be conditional too

Reply:
moved the login to the task where it is neeed